src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -255,7 +255,7 @@ DataView::ActionStatus LiveFunctionsDataView::GetActionStatus(
       action == kMenuActionJumpToMin || action == kMenuActionJumpToMax) {
     if (selected_indices.size() != 1) return ActionStatus::kVisibleButDisabled;
 
-    uint64_t instrumented_function_id = GetInstrumentedFunctionId(selected_indices[0]);
+    uint64_t instrumented_function_id = GetScopeId(selected_indices[0]);
```

Comment:
A lot of the changes in this class assume that a scope_id is equal to the function_id for a function. This holds because of how `NameEqualityScopeIdProvider` is implemented, but it's a strong assumption, and it's hard to follow if you don't know that class. I feel like `AddFunction` should add to `functions_` by scope id now instead of by function_id.

Reply:
Agree. The following code reads quite strange.

Reply:
Good point. As discussed offline, I add FunctionIdToScopeId map (which is identity for now).

For now I keep function_id in a couple of places (action menu and csv export) as these will need to be changed anyway.